wcfsetup/install/files/lib/system/cache/source/RedisCacheSource.class.php

```diff
@@ -46,10 +46,10 @@ public function flush($cacheName, $useWildcard)
         if (isset($parts[1])) {
             if ($useWildcard) {
                 // delete the complete hashset
-                $this->redis->del($this->getCacheName($parts[0]));
+                $this->redis->del($this->getCacheName($parts[0], true));
```

Comment:
A wildcard deletion for the DiskCacheSource will delete both the parameterized as well as the default cache, we must replicate this:
```suggestion
                $this->redis->del(
                    $this->getCacheName($parts[0], true),
                    $this->getCacheName($parts[0]),
                );
```